recipes/confindr/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install olctools && $PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
what is olctools ? if its bioinformatics related you can add it to bioconda before this recipe.
pip shouldn't be used on conda recipes.

Reply:
My apologies, I didn't realize pip shouldn't be used in conda recipes.

olctools is a package that mostly is completely irrelevant - I've integrated the parts of it that were actually necessary into the ConFindr, so I believe it should now be good to go. Let me know if any other changes need to be made.

Thanks!